websites/documentation/gatsby-config.mjs

```diff
@@ -27,6 +27,7 @@ const config = {
             auth0Domain: 'auth.id.commercetools.com',
             auth0ClientId: 'xLk8EDUCc8PKqCbrSJCnuahvn86nEn4z',
             learnApiBaseUrl: 'https://api.learn.commercetools.com',
+            hideLogin: true,
```

Comment:
I can see you have the flag here set as true, so in theory the buttons should be hidden .. but in the preview link they are still visible.

Also, I'm looking at the rest of the code, I can see you are declaring the flag but never used it somewhere or maybe I missed it somewhere

Reply:
Yes because I only implemented the flag in this PR, but I'm not using it yet...but I can do that here already 👍🏼